joomla/joomla-cms
# Subform - Wrong ID generated for the elements of subform when we click on Plus (+) button to add repeated subform #25387

Pull Request for Issue #25387
### Summary of Changes
The value of id attribute of input element is checked for non-word characters and if non-word characters are found then replacing with them with underscore


### Testing Instructions
Check the value of id attribute of input element in repetable subform after clicking the plus (+) button to add new clone of sub-form


### Expected result
The value of id attribute of input element should not contain any non-word character

### Actual result
The non-word characters in the value of id attribute of input element are being replaced with underscore


### Documentation Changes Required
no change required in documentation
